backend/app.ts

```diff
@@ -47,19 +68,20 @@ passport.deserializeUser(async (id: string, done: any) => {
         }
     })
     console.log('found user from cookie')
+    console.log(user)
```

Comment:
debug comment?

Reply:
yes!

Reply:
I think Justin meant that this should be removed